app/models/setting/provisioning.rb

```diff
@@ -21,6 +21,8 @@ def self.load_defaults
         self.set('update_ip_from_built_request', N_("Foreman will update the host IP with the IP that made the built request"), false, N_('Update IP from built request')),
         self.set('use_shortname_for_vms', N_("Foreman will use the short hostname instead of the FQDN for creating new virtual machines"), false, N_('Use short name for VMs')),
         self.set('dns_conflict_timeout', N_("Timeout for DNS conflict validation (in seconds)"), 3, N_('DNS conflict timeout')),
+        self.set('safemode_extra_variables', N_("Extra variables allowed in safe mode"), '', N_('Safemode extra variables')),
```

Comment:
> Does that work?  I wouldn't think that it's either a variable (which would require a code change) or a helper (a top-level method, ditto).  Wouldn't that be whitelisted by creation of Kernel::Jail?

It's a valid example, we actually had to allow it in discovery to make
our examples in the official docs to work.

-- 
Later,
 Lukas #lzap Zapletal